operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -1024,13 +1031,17 @@ message SecretVolume {
 
 // ServiceConfig is described in istio.io documentation.
 message ServiceConfig {
-  google.protobuf.Struct annotations = 1;
-
-  uint32 externalPort = 2;
-
-  string name = 3;
-
-  string type = 18;
+  repeated string ipFamilies = 4;
```

Comment:
These can go to the end?

Reply:
This message isn't being Used. I just removed it.